frontend/kollywood-explorer/style.css

```
@@ -1,755 +1,485 @@
+  --kollywood-dark: #121212;
+  --kollywood-dark-secondary: #1E1E1E;
+  --text-primary: #ffffff;
+  --text-secondary: #cccccc;
+  --accent-red: #D32F2F;
+  --glass-bg: rgba(255, 255, 255, 0.05);
+  --glass-border: rgba(255, 255, 255, 0.1);
+}
+
+/* Base Styles */
+body {
   font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)